source/scss/_library/_objects.sections.scss

```diff
@@ -209,3 +226,103 @@
   }
 
 }
+
+
+/**
+ * Section listing header
+ */
+.c-section__header {
+  padding: $space-double $space;
+  background-color: $c-quaternary;
+  text-align: center;
+  margin-bottom: $space-double;
+
+  @include media('>medium') {
+    padding: $space-triple $space;
+    margin-bottom: $space-quad;
+  }
+}
+.c-section__heading {
```

Comment:
`header/heading`... would you mind renaming this to `.c-section__title`?

Reply:
Changing it to `title` really would fit into the naming conventions I've been using throughout the design system though. Maybe this might help with understanding my reasoning:

- `header` - A container (typically a `<header>` element) that contain elements that typically go in a `<header>` element
- `heading` - Usually a top-level heading element that's reserved for a section or group of components
- `title` - Reserved for an atomic title stemming from a node (article, video, recipe, etc...). Often appears within a section that has a `heading` class.

Reply:
I'm not sure I follow where the distinction is between `heading` and `title` in your breakdown. My main thinking here is autocompletes, slippy fingers, and the considerable overlap in the two words leading to reader/dev confusion. But if this follows a convention you've already established for this project, then it's fine.

Reply:
Does this help?

![img](https://cl.ly/f3a909117ac1/download/Image%2525202019-04-24%252520at%2525203.32.40%252520PM.png)

Reply:
I see. yup :+1: